main_2048.py

```diff
@@ -6,49 +6,51 @@ def play(ia):
         import textual_2048
         dic_move = {0: "g", 1: "d", 2: "h", 3: "b"}

```

Comment:
DO NOT REPLICATE CODE ! 
it s the second time you define this exact same dictionary (and it should be a list...). you can put that in a module (new one or existing one) and use it here.
It is quite common to define constant in upper case like `MOVES = {...}` and then you can use it anywhere, by importing the proper module.